src/app/containers/ChartbeatAnalytics/utils/index.js

```diff
@@ -95,7 +95,7 @@ export const getConfig = ({
       type,
       useCanonical,
       path: currentPath,
-      virtualReferrer: `${origin}/referrer`,
+      virtualReferrer: referrer,
```

Comment:
since we're no longer limiting this to just AMP could we also remove `virtualReferrer: referrer` on line 93 seeing as line 98 would cover for both cases

Reply:
Thanks for the suggestion @rhenshaw56, I've made the changes you suggested